2002-05-31  Mo DeJong  <supermo@bayarea.net>

	* win/tkWinPort.h: Define ALWAYS_SHOW_SELECTION so
	that the text widget selection still works when
	another text widget has the focus. This is required
	for Source-Navigator for tools like a grep/editor
	split window.

Index: tk/win/tkWinPort.h
===================================================================
RCS file: /cvs/src/src/tk/win/tkWinPort.h,v
retrieving revision 1.2
diff -u -r1.2 tkWinPort.h
--- tk/win/tkWinPort.h	10 Sep 2001 01:16:55 -0000	1.2
+++ tk/win/tkWinPort.h	31 May 2002 23:38:52 -0000
@@ -117,6 +117,13 @@
 #define TkpGetNativeAppBitmap(display, name, w, h) None
 
 /*
+ * This macro indicates that entry and text widgets should display
+ * the selection highlight regardless of which window has the focus.
+ */
+
+#define ALWAYS_SHOW_SELECTION
+
+/*
  * timezone et al are already defined in Windows32api headers used by
  * GNU mingw32 port.
  */
